src/distributors/base-installer.ts

```diff
@@ -104,6 +108,7 @@ export abstract class JavaBase {
       if (version[0] >= '0' && version[0] <= '9') {
         version = '>=' + version;
       }
+      access = 'ea';
```

Comment:
I think we should get rid of `else if (version.split('.').length < 3) {`
Also condition `if (version.endsWith('-ea')) {` can be simplified

Reply:
Done. Thank you.